cm/example_multiply/README.md

```diff
@@ -1,3 +1,10 @@
+# Preliminary remarks
+
+The *example_multiply* is complete CM.
+It serves as an example for patners developing others CM.
+
+For more basic CM, see [*example_empty*](../example_empty) CM.
```

Comment:
Make these changes consistent with those made in "cm/example_empty/README.md".

Reply:
I made the change in the last commit